packages/server/src/services/PriceEngine.ts

```
@@ -3,6 +3,10 @@ import { parseConfig, type MarketConfig } from "@percolator/core";
 import { config } from "../config.js";
 import { eventBus } from "./events.js";
 
+// BL2: Extract magic numbers to named constants
+const PENDING_SUB_CLEANUP_INTERVAL_MS = 60_000; // Clean stale subscriptions every 60s
+const PENDING_SUB_MAX_AGE_MS = PENDING_SUB_MAX_AGE_MS; // Stale subscription age threshold
```

Biome (2.3.14)

[error] This variable is used before its declaration.

The variable is declared here:


(lint/correctness/noInvalidUseBeforeDeclaration)